doc/developer/trees.md

```diff
@@ -1,9 +1,15 @@
 # The TreeType policy in mlpack
 
 Trees are an important data structure in mlpack and are used in a number of the
-machine learning algorithms that mlpack implements.  Often, the use of trees can
-allow significant acceleration of an algorithm; this is generally done by
-pruning away large parts of the tree during computation.
+machine learning algorithms that mlpack implements.  mlpack's trees are
```

Comment:
```suggestion
machine learning algorithms that mlpack implements.  Trees in mlpack are
```

I generally dislike 'apostrophe s' in writing but that is a somewhat personal tick. Feel free to ignore.